mitosheet/mitosheet/streamlit/v1/spreadsheet.py

```diff
@@ -83,6 +84,7 @@ def _get_mito_backend(
             *args: Union[pd.DataFrame, str, None], 
             _importers: Optional[List[Callable]]=None, 
             _sheet_functions: Optional[List[Callable]]=None, 
+            _code_options: Optional[CodeOptions]=None,
```

Comment:
For reviewer -- noting here that I'm aiming to allow app developers to specify code options rather than requiring them from the user in the future. This isn't exposed yet, fully.